README.md

```diff
@@ -283,88 +371,74 @@ You may come across some non-English characters. Those apps don't have an Englis
 * [Sparkle](https://sparkleapp.com/) - Pro visual web design.
 * [Figma](https://www.figma.com/) - The collaborative interface design tool, for vector graphics and UI prototyping. ![Freeware][Freeware Icon]![Recommend][tuijian Icon]
 
-#### Prototyping And Mind-mapping Tools
+#### Prototyping and Mind-Mapping Tools
 
-* [Origami Studio](http://origami.design/) -  Tool for designing modern interfaces, built and used by designers at Facebook.
+* [Adobe XD (Experience Design)](http://www.adobe.com/products/experience-design.html) - First all-in-one cross-platform tool for designing and prototyping websites and mobile apps.
+* [Axure RP 8](http://www.axure.com) - Prototypes, specifications and diagrams in one tool. ![hot][hot Icon]
+* [Balsamiq Mockups](https://balsamiq.com/products/mockups/) - Wire-framing tool that helps you work faster and smarter.
 * [Flinto](https://www.flinto.com/) - Quickly create interactive prototypes of mobile, desktop, or web apps. ![Recommend][tuijian Icon]
-* [Kite](https://kiteapp.co/) - Powerful animation and prototyping application for Mac & iOS. ![Recommend][tuijian Icon]
+* [Framer](http://framerjs.com/) - Tool for interactive prototyping. ![star 5][star5 Icon]![hot][hot Icon]
 * [Justinmind](http://www.justinmind.com) - Prototyping platform for web and mobile apps. ![Recommend][tuijian Icon]
+* [Kite](https://kiteapp.co/) - Powerful animation and prototyping application for Mac & iOS. ![Recommend][tuijian Icon]
+* [Lighten](http://lighten.xmind.net/) - The best way to clarify thinking, boost productivity, brainstorm, and visualize concepts.
+* [Marvel](https://marvelapp.com/) - Simple design, prototyping and collaboration.![Freeware][Freeware Icon]
+* [MindNode](http://mindnode.com/) - Mind-mapping software with an emphasis on simplicity and ease-of-use.
 * [MockFlow](https://www.mockflow.com) - Online prototyping suite for web-design and usability testing.
-* [Axure RP 8](http://www.axure.com) - Prototypes, specifications and diagrams in one tool. ![hot][hot Icon]
```

Comment:
@alichtman Your removed Axure.

Reply:
Azure was moved to line 377 under `Prototyping and Mind-Mapping Tools`